temporian/beam/io/test/tensorflow_test.py

```diff
@@ -63,10 +63,10 @@ def test_to_and_from_tensorflow_record(self):
             )
             p.run()
 
-        loaded_evtset = in_process_from_tensorflow_record(
+        loaded_evset = in_process_from_tensorflow_record(
```

Comment:
both `evset` and `evtset` were used, but the former is used way more often and there doesn't seem to be any explanation as to why the extra `t` belongs...